### PR TITLE
update settings for sentry

### PR DESCRIPTION
**Pull request by boney-bun**

### CODE REVIEW

**Comment by boney-bun** [comment]
@Gustry could you kindly review this commit?
I use your code from the projecta :)

**Comment by Gustry** [comment]
Sorry, it's not the only commit, we made a few fixes after the commit from Tim: https://github.com/kartoza/projecta/compare/68b7a62a99d4a02421e67aaddaa14ec848f9d9b6...94ef7b5dd936d6cb09fe248a4a048926901d439e

Sorry it's mixed diff (some other commits are already in the branch), just check the file where Sentry is enabled.

File: deployment/docker/REQUIREMENTS.txt [hunk 1]
```diff
@@ -18,7 +18,8 @@ mezzanine-references==0.1.5
 mezzanine-slides==2.1.2
 OWSLib==0.12
 psycopg2
-raven==5.8.1
+#raven==5.8.1
```
Comment (timlinux) [inline deployment/docker/REQUIREMENTS.txt]: lets delete this rather

File: deployment/docker/REQUIREMENTS.txt [hunk 2]
```diff
@@ -2,7 +2,7 @@ cartridge
 celery
 Django>=1.8,<1.9
 django-compressor
-django-sentry==1.13.5
+#django-sentry==1.13.5
```
Comment (timlinux) [inline deployment/docker/REQUIREMENTS.txt]: lets delete this rather

**Comment by timlinux** [comment]
@boney-bun please also apply the version trick from the changeset @Gustry mentioned above for

deployment/tag_and_deploy_to_staging.sh

**Comment by boney-bun** [comment]
thanks for the feedback @Gustry @timlinux 
i've made the changes based on your input.

File: deployment/tag_and_deploy.sh [hunk 3]
```diff
@@ -4,10 +4,34 @@ echo "Use this to tag a version on your local"
...
-echo "$0 version-1.2.3"
+echo "$0 1.2.3"
 VERSION=$1
-make dbbackup
-make mediasync
-git tag $VERSION
-git push --tags upstream 
-ssh kartoza4 "cd /home/web/kartoza.com/deployment && git fetch --tags && git checkout $VERSION && make build && make kill && make web && make migrate && make collectstatic && make reload"
+echo "Last tag on your local system is:"
+git tag | grep -v version | sort --version-sort | tail -1
```
Comment (Gustry) [inline deployment/tag_and_deploy.sh:10]: It seems you improved it ;-) Thanks

**Review by Gustry — APPROVED** [review]
LGTM

File: deployment/tag_and_deploy.sh [hunk 4]
```diff
@@ -4,10 +4,34 @@ echo "Use this to tag a version on your local"
...
+    # .version is used by Raven/Sentry
+    echo $VERSION > ../django_project/.version
+    git reset HEAD *
+    git add ../django_project/.version
+    git commit -m "bump to version ${VERSION}"
+    git tag $VERSION
+    git push --tags upstream develop
+    # Check it out on the server
+    # No migrations are run - you should do that manually for now
+    ssh kartoza4 "cd /home/web/kartoza.com/deployment && git fetch --tags && git checkout $VERSION && make build && make kill && make web && make migrate && make collectstatic && make reload"
```
Comment (timlinux) [inline deployment/tag_and_deploy.sh:34]: Is live deployed into its own machine now? If yes we should be using kartoza.com everywhere instead of kartoza 4

**Review by timlinux — COMMENTED** [review]
Just one tweak needed

**Pull request merged**